Collaboration.md

```diff
@@ -0,0 +1,155 @@
+
+## #1 Resources & consultation
+
+An existing project is looking for resources and expertise to improve their product's user experience (and potentially design process).
+
+Ways the project can choose to interact with the community:
+
+1. Review the guide to find a direction or solution
+1. Engage in Slack conversations to share ideas and get feedback
+1. Participate in a design review session
+
+Our role in this scenario is to be a helpful partner in providing resources and feedback.
+
+## #2 Solve a specific problem
+
+An existing project has identified a problem they would like support with and is looking to the community for help resolving it.
+
+In addition to the options in #1, the team can also create a project brief to start a deeper collaboration. The brief should include the following:
+
+1. Introduce the team, project, its goals, constraints and design process
+1. Explain the issue to work on
+1. Outline ideas for collaboration
+1. Identify next steps with members of the community
+
+We can share the brief with the community to find contributors.
+
+Solving single problems usually means to work within the current project framework. Bigger discussions around design processes and other improvements can be had, but the scope of work would be to resolve the specific problem that was brought up by the project team.
+
+## #3 Redesign
+
+An existing project is maturing and wants to improve the overall design, which may also means finding a designer and adopting design processes.
+
+The term redesign is often used too quickly. Many times, it is better to systematically improve the existing design than start from scratch. This should be carefully considered.
+
+Steps we can take:
+
+1. Identify goals and vision
+1. Examine the current state of the project
+1. Compare vision to current state
+1. Break down work-to-be-done into manageable parts
+	- Example:
+		1. Refine brand
+		1. User Interface audit
+		1. Create a UI style guide
+		1. Implement UI style guide (without touching interaction design)
+		1. Revise individual application screens and user flows one-by-one
+1. Prioritize
+	- Consider whether to start with the application or the website
+	- Consider level of effort for each change
+	- Consider availability of contributors
+	- Consider time to adopt new workflows
+1. Plan collaboration
+	- Good idea to start with a design review to present the project and create initial interest
+
+Designers who want to tackle redesigns should ensure they have the availability to regularly contribute for longer periods of time. How much time is required depends on the scope. In addition to designers, having a dedicated project manager available to help with planning and communication is also helpful.
+
+## #4 New project
+
+This is an opportunity to adopt a good design process from the start, beginning with researching product concepts, user research, etc. New projects likely also have other foundations to lay, so designers need to be flexible.
```

Comment:
How can you do a user-research when a new project doesn't really have any users in most cases?

Reply:
Maybe target audience is a better term. Main thing is to figure out what project to build before you build it, by becoming familiar with what people's actual needs are.

Reply:
Yup target audience sounds better.